manager-bundle/skeleton/config/config.yaml

```diff
@@ -63,6 +60,11 @@ framework:
 contao:
     preview_script: /preview.php
     messenger:
+        web_worker:
+            transports:
+                - contao_prio_high
+                - contao_prio_normal
+                - contao_prio_low
```

Comment:
Why do we need a separate `contao.messenger.web_worker.transports` parameter? The transports are already registered under `contao.messenger.workers.*.transport`, aren‘t they?

Reply:
Yes we do, this was already discussed with @fritzmg. The `contao.messenger.workers.*.transports` define our cronjob workers. You can add your own transport there or remove those and start real workers using `supervisord` and Co. This is totally independent from the web worker. Because you might want to have the web worker fallback logic also for your own queue. Or you might want to disable it completely but still keep the cronjob messenger logic.

Reply:
I see. 👍